src/contracts/core/DelegationManager.sol

```diff
@@ -442,6 +442,60 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         }
     }
 
+    /**
+     * @notice Called by AVSs to register an operator with the AVS.
+     * @param operator The address of the operator to register.
+     * @param signatureWithSaltAndExpiry The signature, salt, and expiry of the operator's signature.
+     */
+    function registerOperatorWithAVS(address operator, ISignatureUtils.SignatureWithSaltAndExpiry memory signatureWithSaltAndExpiry) external {
+        require(signatureWithSaltAndExpiry.expiry > block.timestamp, "DelegationManager.registerOperatorWithAVS: signature expired");
+        require(operatorSaltIsSpent[operator][signatureWithSaltAndExpiry.salt] == false, "DelegationManager.registerOperatorWithAVS: salt already used");
```

Comment:
very tiny nit, but you can just use the boolean here instead of checking equivalence, like:
` require(!operatorSaltIsSpent[operator][signatureWithSaltAndExpiry.salt], ...)`